apps/mymedicare_cb/models.py

```diff
@@ -61,71 +61,92 @@ def get_and_update_user(
 
     log_dict = {
         "type": "mymedicare_cb:get_and_update_user",
+        "subject": subject,
         "fhir_id": fhir_id,
         "mbi_hash": mbi_hash,
         "hicn_hash": hicn_hash,
         "hash_lookup_type": hash_lookup_type,
         "crosswalk": {},
+        "crosswalk_before": {},
```

Comment:
Added this to the logging to show what the `Crosswalk` record contained before an update.